swift-sdk/Internal/InAppManager.swift

```diff
@@ -613,3 +620,29 @@ extension InAppManager: InAppNotifiable {
         return result
     }
 }
+
+extension InAppManager: InAppDisplayChecker {
+    func isOkToShowNow(message: IterableInAppMessage) -> Bool {
+        guard !isAutoDisplayPaused() else {
```

Comment:
nit: Not needed for +1. why not call `autoDisplayPaused` variable? 

Style-wise, if `isAuthoDisplayPaused` was a property rather than a variable then it would make sense to call `isAuthoDisplayPaused`  it rather than the function.